packer/install-dependencies.sh

```diff
@@ -364,15 +364,13 @@ echo "  - busybox" >> ${RELEASE_NOTES_FILEPATH}
 K8S_VERSIONS="
 1.15.0
 1.14.4
+1.14.4-azs
```

Comment:
should we update the Azure Stack docs list of supported versions?

Reply:
It completely slipped from my mind. Let me update the docs as well.

Reply:
Update the PR with doc.